packages/maskbook/src/plugins/Furucombo/UI/FurucomboView.tsx

```diff
@@ -0,0 +1,165 @@
+import { makeStyles } from '@masknet/theme'
+import { useChainId } from '@masknet/web3-shared'
+import { Card, CardContent, CardActions, Tabs, Tab, Typography, Link, Paper } from '@material-ui/core'
+import { useState } from 'react'
+import { FurucomboIcon } from '../../../resources/FurucomboIcon'
+import { MaskTextIcon } from '../../../resources/MaskIcon'
+import { useI18N } from '../../../utils/i18n-next-ui'
+import { useFetchPools } from '../hooks/usePool'
+import type { Investable } from '../types'
+import { InvestmentsView } from './InvestmentsView'
+import { Poolview } from './PoolView'
+
+const useStyles = makeStyles()((theme) => ({
+    root: {
+        fontFamily: 'Muli,Helvetica,Arial,"sans-serif"',
```

Comment:
why set font family?

CC @Jack-Works

Reply:
I used the font from the furucombo app to get as close as possible to their design. I can remove it if you want...

Reply:
@guanbinrui